.github/actions/tests/run-e2e-tests/action.yml

```diff
@@ -37,6 +37,18 @@ runs:
               --config=tests/e2e-pw/${{ inputs.playwright-config }} \
               ${{ inputs.tests }}
 
+        - name: Run E2E tests on chrome-new-editor.
```

Comment:
`chrome-new-editor` and the usage of `new-editor` are very generic to me.
This should be changed to either just `product-editor` or `new-product-editor`

Also does this still get used?

Reply:
> chrome-new-editor and the usage of new-editor are very generic to me.
This should be changed to either just product-editor or new-product-editor

Done in commit https://github.com/woocommerce/woocommerce/pull/41601/commits/81a09a1b8cecd4595664451ab02a28e1c8f08efe

> Also does this still get used?

We use them in the [smoke-test-release](https://github.com/woocommerce/woocommerce/blob/trunk/.github/workflows/smoke-test-release.yml#L62) and [smoke-test-daily](https://github.com/woocommerce/woocommerce/blob/trunk/.github/workflows/smoke-test-daily.yml#L238) workflows.